site/pages/create-disruption/[disruptionId].page.tsx

```diff
@@ -322,7 +331,7 @@ const CreateDisruption = (props: DisruptionPageProps): ReactElement => {
                             </Fragment>
                         ))}
                         <div className="flex pb-8">
-                            <div>
+                            <div className={`${dateColumnError ? "min-w-[300px] pr-1" : "pr-1"}`}>
```

Comment:
As per other comment, bump this up to 320px or something similar if min width is added to the input. Repeat elsewhere in this file where min width is 300px

Reply:
done